dlx_rest/static/js/import.js

```diff
@@ -205,23 +234,30 @@ export let importcomponent = {
                         let fatalErrors = []
 
                         if (jmarc.fields.length > 0) {
-                            jmarc.symbolInUse().then( symbolInUse => {
+                            jmarc.symbolInUse().then(symbolInUse => {
                                 if (symbolInUse) {
                                     this.issues += 1
-                                    validationErrors.push({"message": "Duplicate Symbol Warning: The symbol for this record is already in use."})
+                                    validationErrors.push({ "message": "Duplicate Symbol Warning: The symbol for this record is already in use." })
                                 }
                             });
 
-                            for (let field of jmarc.fields.filter(x => ! x.tag.match(/^00/))) {
+                            for (let field of jmarc.fields.filter(x => !x.tag.match(/^00/))) {
                                 for (let subfield of field.subfields.filter(x => 'xref' in x)) {
                                     if (subfield.xref instanceof Error) {
                                         // unresolved xrefs are set to an Error object
-                                        fatalErrors.push({"message": `Fatal: ${field.tag}$${subfield.code} ${subfield.xref.message}: ${subfield.value}`})
+                                        fatalErrors.push({ "message": `Fatal: ${field.tag}$${subfield.code} ${subfield.xref.message}: ${subfield.value}` })
                                     }
                                 }
                             }
 
-                            this.records.push({"jmarc": jmarc, "mrk": mrk, "validationErrors": validationErrors, "fatalErrors": fatalErrors, "checked": false})
+                            // Set a field indicating the record was imported
```

Comment:
Are we sure we want to set this as a value inside the record data?

Reply:
The request was to use a field to indicate the record was imported. We can present it and see if that suffices.